### PR TITLE
update styles, validation JobPostForm

### PR DESCRIPTION
**Pull request by MykhailoKutsyi**
update styles to antd styles
updated form validation schema

image

All data collected to object
all logic made with useForm and also Controller  from 'react-hook-form'
 
image

### CODE REVIEW

**Comment by lyudad** [comment]
Make sure that you use the same inputs through all projects, I think it's different from sign up form

**Review by lyudad — APPROVED** [review]

**Comment by MykhailoKutsyi** [comment]
added second job post form page

For now, it is realized that with useState, we have one page and change content on it with useState, how do you think it can be?  When submit first form page, if all entered data ok → setState change → shown second form. 
Also, for second form added second shema and contol.

If we need second page without first I can rewrite to component props, for example,

First form
image

Second form
image

**Comment by lyudad** [comment]
I would go with redux

**Comment by MykhailoKutsyi** [comment]
Hello there,

So, separated one page on two (discussed with team at the meet)
job-post-first-page
job-post-second-page

Updated Input to StyledInput.

Also now selectors options come from back, so I rewrited this components and add this feature with hook useProperties().

File: libs/components/src/lib/JobPostFormFirstPage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+import { InputsValues, schema } from '../constants';
+
+import { StyledErrorMessage } from './styles';
+
+export function JobPostFormFirstPage() {
+  const { categories, skills, getOptionsForSelectWithId } = useProperties();
+  const navigate = useNavigate();
+  const { t } = useTranslation();
+  const { Title, Text } = Typography;
```
Comment (lyudad) [inline libs/components/src/lib/JobPostFormFirstPage/index.tsx:22]: move out of component, you now initialise constant with each rerender

Comment (MykhailoKutsyi) [reply libs/components/src/lib/JobPostFormFirstPage/index.tsx:22]: yes, thanks

**Pull request merged**